src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetXml.scala

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.deposit.docs
+
+import nl.knaw.dans.easy.deposit.docs.DatasetMetadata.DateQualifier.{ available, created }
+import nl.knaw.dans.easy.deposit.docs.DatasetMetadata._
+
+import scala.util.Try
+import scala.xml.{ Attribute, Elem, Null, PrefixedAttribute }
+
+object DatasetXml {
+  private type MaybeAttribute = Option[Attribute]
+
+  private val otherDates = DateQualifier.values.filter(d => d != created && d != available).toSeq
```

Comment:
`created` -> `DateQualifier.created`, analogous for `available`
would that make it clearer?

Reply:
```scala
  private val otherDateQualifiers = DateQualifier.values.filter(qualifier => 
    qualifier != DateQualifier.created && qualifier != DateQualifier.available
  ).toSeq
```

Reply:
can you use `filterNot` here? maybe that would better signify what you're doing?